src/OpenLoco/src/Paint/Paint.cpp

```diff
@@ -1073,6 +1085,13 @@ namespace OpenLoco::Paint
                 return true;
             }
         }
+        if ((viewFlags & Ui::ViewportFlags::seeThroughBridges) != Ui::ViewportFlags::none)
+        {
+            if (isTypeCullableBridge(ps.type))
```

Comment:
i would be inclined to add this to the outer `if` to reduce nesting, but it's all the same in the end

Reply:
Done